_posts/2021-04-02-cultural-competency-in-archives-planning-an-education-campaign.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Learning Together: Cultural Competency in Archives - Planning an Education Campaign"
+date: 2021-04-02T11:10:00
+author:
+- Amy Berish
```

Comment:
I think this is missing an indent.

Reply:
Actually, I'd put this on a single line. `author: Amy Berish`